charts/xui-approve-org/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-name: xui-approve-org
+name: xuiao-webapp
```

Comment:
The chart name must match the chart directory. If this must be xuiao-webapp then the dir name must be this.